plugins/jenkins/test/models/samson/jenkins_test.rb

```diff
@@ -86,4 +97,36 @@ def stub_get_build_id_from_queue(build_id)
       jenkins.job_url(96).must_equal "https://jenkins.zende.sk/job/rdhanoa_test_project/96/"
     end
   end
+
+  describe "#build_with_env_flags" do
+    before(:each) do
+      stub_crumb
+      stub_job_detail
+    end
+
+    it "check for both emails when deployer has buddy" do
+      deploy.stubs(:buddy).returns(buddy)
+      stub_build_with_parameters("emails": "super-admin@example.com,deployerbuddy@example.com")
+      stub_get_build_id_from_queue(1)
+      jenkins.build.must_equal 1
+    end
+
+    it "checks for committers email when JENKINS_NOTIFY_COMMITTERS is set" do
```

Comment:
We also need a test with `stub_add_changeset ` but no env var to show that committers emails are not included when the env var is not set.